src/Components/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import LoginArticle from "./Components/LoginArticle";
+import RegisterArticle from "./Components/RegisterArticle";
+import ErrorTextBox from "./Components/ErrorTextBox";
+import { config } from "./Components/config";
+
+class LoginInput extends Component {
+  constructor() {
+    super();
+    this.state = {
+      errorMessage: "asdasd!!",
+      userID: "",
+      loginPassword: "",
+      email: "",
+      regPassword: "",
+    };
+  }
+
+  sendUpSign = (key) => {
+    let jsonBody, configtemp;
+    key === "login"
+      ? (jsonBody = {
+          userID: this.state.userID,
+          loginPassword: this.state.loginPassword,
+        }) && (configtemp = `${config}/signin`)
+      : (jsonBody = {
+          email: this.state.email,
+          regPassword: this.state.regPassword,
+        }) && (configtemp = `${config}/signup`);
+
+    fetch(configtemp, {
+      method: "POST",
+      body: JSON.stringify(jsonBody),
+    })
+      .then((res) => res.json())
+      .then((res) => this.setState({ errorMessage: res }));
+  };
+
+  onClickHandler = (event) => {
+    this.sendUpSign(event);
+  };
+
+  onChangeHandler = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  render() {
+    const {
+      errorMessage,
+      userID,
+      loginPassword,
+      email,
+      regPassword,
+    } = this.state;
+
+    return (
+      <>
+        {errorMessage !== "" ? <ErrorTextBox printText={errorMessage} /> : ""}
```

Comment:
```suggestion
        {errorMessage && <ErrorTextBox printText={errorMessage} /> }
```
- ""의 불리언 값은 false입니다. false가 아닌 경우이므로 true겠죠?
- true일 때에 대해서만 조건부 렌더링을 한다면 &&를 사용해주세요